app/presenters/order_detail_presenter.rb

```diff
@@ -49,6 +53,12 @@ def survey_url
     survey_completed? ? external_service_receiver.show_url : ""
   end
 
+  def wrapped_total
```

Comment:
Why do we need to do this wrapping here? I get that the object might need the `PriceDisplayment`, but why does it need a `reload`?